e2e/test-subjects/shell-next/pages/react-error.tsx

```diff
@@ -0,0 +1,22 @@
+import type { NextPage } from "next";
+import { Host } from "@leanjs/next";
+import { useRouter } from "next/router";
+import { CustomError } from "../components/CustomError";
+
+const ReactError: NextPage = () => {
+  const {
+    query: { customError },
+  } = useRouter();
+
+  return (
+    <>
+      <h1>React Error page</h1>
+      <Host
+        errorComponent={customError ? CustomError : undefined}
```

Comment:
Could simplify it to `errorComponent={CustomError}` and delete the above?:
```
const {
    query: { customError },
  } = useRouter();
```
and also in the test `cy.visit("http://localhost:44443/react-error?customError=true");` simplify it to `cy.visit("http://localhost:44443/react-error");`

Reply:
resolved in latest commit